users.py

```diff
@@ -35,3 +35,6 @@ def addWorkflow(self, workflow):
         else:
             self.workflows.append(workflow)
 
+    # TODO - log
+    # TODO - screenshots
+    # TODO - think
```

Comment:
users should be able to log their results to file and take screenshots. Also, once we implement user skill level, a nice way to safely abstract that away is to tell a user to think. so if a task has a 2 seconds "think" period, we call `user.think(2)` and the thing method does something like `time.sleep(self.skill * duration)`.